.github/workflows/ai-pr-reviewer.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   review:
     runs-on: ubuntu-latest
     steps:
-      - uses: coderabbitai/ai-pr-reviewer@latest
+      - uses: Vagif12/ai-pr-reviewer@latest
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `coderabbitai/ai-pr-reviewer@latest` to `Vagif12/ai-pr-reviewer@latest` might cause issues if the new repository does not have the same functionality as the old one. Ensure that `Vagif12/ai-pr-reviewer` is a reliable source and has the necessary features for your project.
```diff
- uses: Vagif12/ai-pr-reviewer@latest
+ uses: coderabbitai/ai-pr-reviewer@latest
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->